deploy/gcp/README.md

```diff
@@ -0,0 +1,97 @@
+# Deploy TiDB Operator and TiDB cluster on GCP GKE
+
+## Requirements:
+* [gcloud](https://cloud.google.com/sdk/install)
+* [terraform](https://www.terraform.io/downloads.html)
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl) >= 1.11
+* [helm](https://github.com/helm/helm/blob/master/docs/install.md#installing-the-helm-client) >= 2.9.0
+* [jq](https://stedolan.github.io/jq/download/)
+
+## Configure gcloud
+
+https://cloud.google.com/sdk/docs/initializing
+
+## Setup
+
+The default setup will create a new VPC, two subnetworks, and an f1-micro instance as a bastion machine. The GKE cluster is created with the following instance types as worker nodes:
```

Comment:
The default setup exceeds the default cpu quota of a GCP project. There should be a note about this.

Reply:
Added a note that addresses this.